api/controllers/DashboardController.js

```diff
@@ -125,19 +120,17 @@ const DashboardController = {
             });
     },
 
-    _chooseStoredProcedure(secondarySortOrder, electronicEnabled = false) {
-        const procedureToUse = electronicEnabled
-            ? 'dashboard_data_eapp'
-            : 'dashboard_data';
+    _chooseStoredProcedure(secondarySortOrder) {
+        const procedureToUse = 'dashboard_data_eapp';
```

Comment:
Hey Ali, this is a really fair point, I left it because i thought it was a bit of a faff to change stored functions in the DB but it does make sense to change.

Reply:
ah ok, I overlooked that these were stored procedures.  Yep, agreed in that case, too much of a faff to change this particular area but there are also other areas in regular JS code that you may want to consider removing the eApp suffix on in a number of areas throughout.